### ISSUE TITLE
HTML files are not valid

### ISSUE THREAD

**Issue by fenuks** [Michael-F-Bryan/mdbook-epub#29]
Generated HTML files inside epub file are not valid. They don't have DOCTYPE, nor required tags. I generated books with intent to read them in [KOReader](https://github.com/koreader/koreader), and it cannot load them. Calibre book reader, or okular are more forgiving.

Below sample of few first lines of HTML document from generated ‘Rust by Example’ epub:

```html
<h1>Rust by Example</h1>
<p><a href="https://www.rust-lang.org/">Rust</a> is a modern systems programming language focusing on safety, speed,
and concurrency. It accomplishes these goals by being memory safe without using 
garbage collection.</p>
```

Using mdbook-epub version 0.3.0.

**Comment by mabl** [comment]
I got KOreader to parse the file by converting it from EPUB to EPUB.

**Comment by mabl** [comment]
I got it fixed by building against the master of epub-builder. See also https://github.com/lise-henry/epub-builder/issues/8

**Comment by mabl** [comment]
This should be fixed with epub-builder 0.4.6.

**Comment by fenuks** [comment]
Thank you, I can confirm it works fine now.
